### ISSUE TITLE
Compilation errors on MacOS Catalina

### ISSUE THREAD

**Issue by aleSuglia** [microsoft/jericho#37]
# Issue
I am having issues installing Jericho using pip (that is issued when trying to install TextWorld). The compilation error I get is:

```
src/ztools/infodump.c:479:5: error: implicit declaration of function 'configure_inform_tables' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
        configure_inform_tables(obj_data_end, &inform_version, &class_numbers_base, &class_numbers_end,
        ^
    1 warning and 1 error generated.
    make: *** [src/ztools/infodump.o] Error 1
    make: *** Waiting for unfinished jobs....
```
You can see the full stacktrace [here](https://bpa.st/BL3A).

# Setup

- Python 3.8.5 (using Anaconda)
- MacOS Catalina (10.15.7)
- Apple clang version 12.0.0 (clang-1200.0.32.21)  
  
# Question

Is there any plan to support MacOS as well or this package is meant to be installed on Linux only?

**Comment by MarcCote** [comment]
@aleSuglia we do support MacOS. We are testing using this build system [MacOS 10.15](https://github.com/actions/virtual-environments/blob/main/images/macos/macos-10.15-Readme.md). According to the list of available software, we are not using Apple clang 12, which seems to be the one causing a problem (see similar issue: https://github.com/kaldi-asr/kaldi/issues/4155).

Maybe there's a way to tell the compiler to keep it as a warning instead of an error (i.e., `-Wimplicit-function-declaration`). I'll try to patch it today.

**Comment by MarcCote** [comment]
@aleSuglia I made a fix (see PR #38). Can you try to see if that works for you?
```
pip install https://github.com/MarcCote/jericho/archive/fix_37.zip
```
Thanks.

**Comment by aleSuglia** [comment]
Mmm looks like that it didn't fix: https://bpa.st/WRQQ

**Comment by MarcCote** [comment]
You are right. It seems the compiler flag wasn't used when compiling the main interface :-o. You can try again.
```
pip install https://github.com/MarcCote/jericho/archive/fix_37.zip
```

**Comment by aleSuglia** [comment]
All good it works 👍

**Comment by MarcCote** [comment]
Thanks for confirming. @mhauskn  ^

**Comment by mhauskn** [comment]
Merged #38 into master. Thanks @MarcCote for the fixes!

**Comment by aleSuglia** [comment]
Closing this one. Thanks @MarcCote for the prompt fix!
